src/components/notifications/GistExportNotification.jsx

```diff
@@ -15,5 +15,6 @@ export default function GistExportNotification(props) {
 
 GistExportNotification.propTypes = {
   payload: PropTypes.object.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Hmm, good question—the reason I left it in there is that the abstract `ExportNotification` component type does take a `type` key. But it occurs to me that neither `PropTypes` nor `eslint-plugin-react` complains if you pass _extra_ propTypes to a component (although I wish they did!) so I guess there’s no reason to keep this in.